script/test.js

```diff
@@ -51,6 +51,10 @@ if (process.env.HARMONY) {
   nodeArgs.push("--harmony")
 }
 
+if (process.version.startsWith("v10.") || process.version.startsWith("v11.")) {
+  nodeArgs.push("--experimental-worker")
+}
+
```

Comment:
☝️ You can use `semver` for this check (it's a dev dep we use for just this kind of sniffing for our tests).
```js
const SemVer = require("semver")
// then later
SemVer.satisfies(process.version, ">10.5.0")
```